README.md

```diff
@@ -169,26 +394,25 @@ Your `Staker UI` tab should be almost done and working at this point.
 
 ---
 
-### ⚔️ Side Quests
+## ⚔️ Side Quests
 
 - [ ] Can `execute()` get called more than once, and is that okay?
-- [ ] Can you stake and withdraw freely after the `deadline`, and is that okay?
-- [ ] What are other implications of _anyone_ being able to withdraw for someone?
+- [ ] Can you contribute and withdraw freely after the `deadline`, and is that okay?
 
 ---
 
 ### 🐸 It's a trap!
 
 - [ ] Make sure funds can't get trapped in the contract! **Try sending funds after you have executed! What happens?**
-- [ ] Try to create a [modifier](https://solidity-by-example.org/function-modifier/) called `notCompleted`. It will check that `ExampleExternalContract` is not completed yet. Use it to protect your `execute` and `withdraw` functions.
+- [ ] Update the [modifier](https://solidity-by-example.org/function-modifier/) called `notCompleted`. It should check that `FundingRecipient` is not completed yet. Use it to protect your `execute`, `contribute` and `withdraw` functions by throwing a new custom error if it has already been completed.
 
 ### ⚠️ Test it!
 
-- Now is a good time to run `yarn test` to run the automated testing function. It will test that you hit the core checkpoints. You are looking for all green checkmarks and passing tests!
+- Now is a good time to run `yarn test` to run the automated testing for everything you have done. It will test that you hit the core checkpoints. You are looking for all green checkmarks and passing tests!
 
 ---
 
-## Checkpoint 4: 💾 Deploy your contract! 🛰
+## Checkpoint 5: 💾 Deploy your contract! 🛰
 
 📡 Edit the `defaultNetwork` to [your choice of public EVM networks](https://ethereum.org/en/developers/docs/networks/) in `packages/hardhat/hardhat.config.ts`
```

Comment:
currently we support only [these](https://github.com/austintgriffith/speedrun-grader/blob/main/utils/supported-chains.js) networks. So I think it's better to suggest to use sepolia or optimismSepolia for now as in the hint below